proposals/unified-show-diff.md

```diff
@@ -0,0 +1,134 @@
+# Title: Unified show/diff command for all output types
+
+- Enhancement Proposal PR: (leave this empty)
+- Contributors: dberenbaum
+
+# Summary
+
+Establish a unified command to combine and `show` or `diff` all outputs of
+specified types, including:
+
+* Metrics
+* Plots
+* Images (depends on https://github.com/iterative/dvc/discussions/5681)
+
+These would all be put under a single `dvc outputs show/diff` command.
```

Comment:
I like `dvc out(put)s`. What I suggested originally was `dvc compare` (since `diff` is taken) but that only consolidated metrics/plots/experiments diff` commands. If we decide to keep 2 top commands maybe we have have both `outs` (merged `show`s) and `compare` (merged `diff`s).